### PR TITLE
2277 basic graphql service

### PR DESCRIPTION
**Pull request by mikhail-balabaev**
I created a basic GraphQL service [AB#2277](https://deloittedigitalat.visualstudio.com/38398c5c-11b2-4e64-98b5-5af5e65d6e82/_workitems/edit/2277):
- added type-graphql schemas and resolvers
- added tests for schemas and resolvers
- added Dockerfile
- added GraphQL queries in Readme

### CODE REVIEW

File: data-service/README.md [hunk 1]
```diff
@@ -41,3 +41,76 @@ To run Prettier
 ```
 npm run pretty-quick
 ```
+
+## Build Docker container
+
+```
+docker build -t core-banking/data-service .
+```
+
```
Comment (theqajedi) [inline data-service/README.md:50]: Can you also please add details about ACCOUNTS_HOST and TRANSACTIONS_HOST env variables in readme?

**Review by theqajedi — APPROVED** [review]

File: data-service/README.md (hunk 1 above)
Comment (aberke220) [reply data-service/README.md:50]: + PORT as well pls :) and any other env var needed to configure the service.

File: data-service/README.md [hunk 2]
```diff
@@ -41,3 +41,76 @@ To run Prettier
...
+### query variables
+
+```
+{
+  "id":"B12456",
+  "accountId": 12286356
+}
+```
```
Comment (aberke220) [inline data-service/README.md:116]: Could you please add usage example to illustrate how to use this service?
Just a sample request and response to see how the query works. Thanks!

File: data-service/README.md (hunk 1 above)
Comment (theqajedi) [reply data-service/README.md:50]: added in the new PR
https://github.com/DeloitteDigitalAT/core-banking/pull/25

**Pull request merged**